src/vs/code/node/cliProcessMain.ts

```diff
@@ -140,8 +134,18 @@ class Main {
 								return TPromise.wrapError(new Error(`${notFound(id)}\n${useId}`));
 							}
 
-							console.log(localize('foundExtension', "Found '{0}' in the marketplace.", id));
```

Comment:
What happens if the extension is not installed. I assume it will throw an exception now as it is not being checked.

There are three flows:

1. Extension is not installed
2. Extension is installed but update required
3. Extension is installed and no update required.

I do not see where the first case is handled here

Reply:
Done.